test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
I think putting more assumptions into collins to address [mis]configuration of servers during lshw upload is a road we do not want to go further down. IMHO, speed stepping should be disabled by genesis (if it isnt already) before lshw collection, as a workaround. This definitely merits more discussion though, because of the assumptions baked into collins. Could you open an issue and tag me in it? Thanks!